tests/ha/filesystem.pm

```diff
@@ -66,17 +69,19 @@ sub run {
     # DLM process needs to be started
     ensure_process_running 'dlm_controld';
 
-    # ocfs2 package should be installed by default
-    if ($fs_type eq 'ocfs2') {
-        die 'ocfs2-kmp-default kernel package is not installed' unless is_package_installed 'ocfs2-kmp-default';
-    }
-
+    # gfs2-utils is not installed by default, so we need to install it if needed
+    zypper_call 'in gfs2-utils gfs2-kmp-default' if ($fs_type eq 'gfs2' && is_sle('<16'));
```

Comment:
why the `&&` is needed? Looks like other places of this PR follow the logic to have `$fs_type` assigned at the beginning and then calculate everything else from its value

Reply:
At this time it's not clear whether we need to install these packages explicitly in 16 or not, but we do need to install them explicitly in 15 or lower. I'll add a comment to make this more clear.

Reply:
Done. Added a comment with more details in the line above.